crates/sui-graphql-rpc/src/types/address.rs

```diff
@@ -17,6 +17,8 @@ use super::{
     transaction_block::{TransactionBlock, TransactionBlockFilter},
 };
 
+/// The 32-byte address that is either an account address
+/// (derived from a particular signature scheme) or an object ID.
```

Comment:
```suggestion
/// The 32-byte address that is either an account address (corresponding to a public key).
```

We don't use this `Address` type to refer to ObjectIDs, it's only the addresses that correspond to public keys.